openslides/assignment/forms.py

```diff
@@ -8,12 +8,12 @@
 
 
 class AssignmentForm(CssClassMixin, forms.ModelForm):
-    posts = forms.IntegerField(
+    member_count = forms.IntegerField(
```

Comment:
At the moment this field decides whether to use a yes-no-abstain poll or not if the config is set to the respective value. I this (default) use case if an assignment has 8 members of which 4 have to be elected you have to set the field to 4 not to 8 to get a normal poll if there are 5 candidates. So the name of the field is not correct with the current implementation.